<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cubeLoader</class>
 <widget class="QDialog" name="cubeLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>169</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>cubeLoader</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Type of cube</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cubeComboBox">
       <item>
        <property name="text">
         <string>MIRI</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>File Path: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="subbandComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>1 Short</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 Long</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 Short</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 Long</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 Short</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 Long</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 Short</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 Long</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="acceptButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Subband associate with</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="selectFileButton">
       <property name="text">
        <string>Select .fits file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="selectDirectoryButton">
       <property name="text">
        <string>Select directory</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLabel" name="filePathLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="filterLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>*.fits</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="filterLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Filter text</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="customSubbandCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Use custom subband instead of default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>cubeLoader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>cubeLoader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
